src/systemd/sd-dhcp-server.h

```diff
@@ -31,6 +31,10 @@ _SD_BEGIN_DECLARATIONS;
 
 typedef struct sd_dhcp_server sd_dhcp_server;
 
+enum {
+        SD_DHCP_SERVER_EVENT_LEASE_CHANGED,
```

Comment:
This should have an explicit value assigned. See SD_JOURNAL_LOCAL_ONLY in sd-journal.h.

Reply:
Do you think it should be bitflags?

Reply:
done